tests/test_events.py

```
@@ -1,3 +1,8 @@
+from dotenv import load_dotenv
+load_dotenv(dotenv_path=".env.local")
+import sys
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)